modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/NodeViewRoutesBase.php

```diff
@@ -89,4 +94,36 @@ protected function getNode(): ?NodeInterface {
     return $event->getNode();
   }
 
+  /**
+   * Gets introduction data from the node.
+   *
+   * @param string $field_name
+   *   Node's field name where data is taken from.
+   *
+   * @return array
+   *   Inline template if data exists or NULL otherwise.
+   */
+  protected function getPageHeaderIntroduction(string $field_name = 'oe_summary'): array {
```

Comment:
```suggestion
  protected function getIntroductionMetadata(string $field_name): array {
```
I propose these two changes:
- Not including PageHeader in the name as it's already in the class name
- removing  this default parameter value, it will be clearer if the extending class explicitly request the field it wants to render.

Reply:
so I'm sending the parameter
`$metadata = [
      'introduction' => $this->getIntroductionMetadata('oe_summary'),
    ];`